sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -117,6 +120,8 @@ object GpuHashJoin extends Arm {
         if (keyDataTypes.exists(_.isInstanceOf[StructType])) {
           meta.willNotWorkOnGpu(s"$joinType joins currently do not support with struct keys")
         }
+      case ExistenceJoin(_) =>
+        // TODO enabling existence join
```

Comment:
This TODO needs to be addressed.  If the join has a condition then it needs to be applied when on the GPU, and if it cannot be processed by the GPU then we need to fallback to the CPU.  Tests need to be added for these cases.  If this is not going to be addressed in this PR then this needs to do the safe thing, which is fallback to the CPU.  As it is now, it will allow some existence joins that cannot be processed on the GPU to be on the GPU which will either result in a crash or silent data corruption.